### PR TITLE
Feature/#27 login

### PR DESCRIPTION
**Pull request by ikim23**
### Done:
- backend service, that takes login & password and return JWT token (is not connected to MongoDB)
- frontend login page
- persisting of JWT token in local storage
- securing react app routes (timetable can be accessed only when user is logged in & other pages will automatically redirect to /timetable route when user is logged in)
- user reducer unit tests

### Needs to be implemented (issues created & linked here):

- wire up backend service to database (#71)
- create "auth flow" in react using redux-sagas (automatic renew of JWT token/redirect to login page when token is about to expire) (#72)

### CODE REVIEW

**Comment by ikim23** [comment]
@marektavac I does exist. See [this](https://github.com/fri-team/Swapify/pull/75/files#diff-10efdddcec23606d3b2229d68c068bd7R20).

**Comment by ikim23** [comment]
OK. I forgot to update the CI script on this branch. I fixed it and it fails now, as it should.

**Comment by ikim23** [comment]
Unit tests fixed.

File: WebAPI/Models/AuthUser.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using System;
+using System.IdentityModel.Tokens.Jwt;
+
+namespace WebAPI.Models
+{
+    public class AuthUser
```
Comment (marektavac) [inline WebAPI/Models/AuthUser.cs]: What about use the whole name for class?

**Pull request merged**